desktop/release-notes.md

```diff
@@ -24,13 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.16.3
+
+{% include release-date.html date="2023-01-26" %}
```

Comment:
Delta updates are still being created, we should probably postpone releasing until tomorrow.
```suggestion
{% include release-date.html date="2023-01-27" %}
```